cli/azd/resources/scaffold/templates/resources.bicept

```diff
@@ -717,19 +728,11 @@ module {{bicepName .Name}} 'br/public:avm/res/app/container-app:0.8.0' = {
             value: {{bicepName .Name}}Identity.outputs.clientId
           }
           {{- end}}
-          {{- if (and .AzureEventHubs .AzureEventHubs.AuthUsingConnectionString) }}
+          {{- if (or (and .AzureEventHubs .AzureEventHubs.AuthUsingConnectionString) (and .AzureEventHubsForKafka .AzureEventHubsForKafka.AuthUsingConnectionString)) }}
           {
-            name: 'SPRING_CLOUD_AZURE_EVENTHUBS_CONNECTIONSTRING'
+            name: 'SPRING_CLOUD_AZURE_EVENTHUBS_CONNECTION_STRING'
             secretRef: 'event-hubs-connection-string'
           }
-          {
-            name: 'SPRING_CLOUD_AZURE_EVENTHUBS_CREDENTIAL_MANAGEDIDENTITYENABLED'
-            value: 'false'
-          }
-          {
-            name: 'SPRING_CLOUD_AZURE_EVENTHUBS_CREDENTIAL_CLIENTID'
-            value: ''
-          }
```

Comment:
Same here. These values can be used to override the values specified in `application.yml`.